packages/ecc-utils-design/src/components/form/form.ts

```diff
@@ -526,6 +526,16 @@ export default class EccUtilsDesignForm extends LitElement {
       return html` ${this.renderSuccessTemplate()} `;
     }
 
+    const toggleButtonState = () => {
```

Comment:
Hey are these changes from another issues that might have slipped in here :sweat_smile:

Reply:
Yeah, I added them. since it was just a minor modification in 1 file.